lib/addRemove.js

```diff
@@ -64,7 +64,7 @@ function addAsync(version, useNow) {
 			let versionDir = nvsUse.getVersionDir(version);
```

Comment:
LOL, I didn't answer because I didn't want to get into that debate. :)

I just wish GitHub had better support for tabs.